gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/ApiService.java

```diff
@@ -44,7 +45,7 @@ public interface ApiService {
 
     Set<ApiEntity> findByEnvironmentAndIdIn(ExecutionContext executionContext, Set<String> ids);
 
-    Set<ApiEntity> findAllLightByEnvironment(ExecutionContext executionContext);
+    Stream<ApiEntity> findAllLightByEnvironment(ExecutionContext executionContext);
```

Comment:
could be removed no ? 
because is only used here for api Ids : https://github.com/gravitee-io/gravitee-api-management/blob/d1fb38bde395cc2005d0d6f5cfe6fd6db16e91eb/gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/upgrade/DocumentationSystemFolderUpgrader.java#L80